File: tema8/ListadoPersonasCRUD/src/Domain/Entities/Departamento.ts

export class Departamento {
  private _idDepartamento: number;
  private _nombreDepartamento: string;

  constructor(idDepartamento: number, nombreDepartamento: string) {
    this._idDepartamento = idDepartamento;
    this._nombreDepartamento = nombreDepartamento;
  }

  public get idDepartamento(): number {
    return this._idDepartamento;
  }

  public get nombreDepartamento(): string {
    return this._nombreDepartamento;
  }
}

File: SGE/unit10/ListaPersonaCRUD/Domain/entities/Departamento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.entities
{
    public class Departamento
    {
        // Atributos privados según el esquema
        private int _id;
        private string _nombre;

        // Propiedades públicas para el acceso (GETTERS y SETTERS)
        public int Id { get => _id; set => _id = value; }
        public string Nombre { get => _nombre; set => _nombre = value; }

        // Constructor
        public Departamento(int id, string nombre)
        {
            _id = id;
            _nombre = nombre;
        }
    }
}